odl/contrib/theano/README.md

```diff
@@ -0,0 +1,17 @@
+# Theano
+
+This package contains ODL functionality related to [Theano](http://www.deeplearning.net/software/theano/).
+
+## Content
+
+* `TheanoOperator` in [layer.py](layer.py) wraps an ODL operator into a Theano operator.
+  This allows using arbitrary ODL operators inside theano computational graphs.
```

Comment:
I'd capitalize Theano, except when used as package name \`\`theano\`\`
Across the board.